pages/contact/_form.html

```diff
@@ -16,8 +16,6 @@
         {% include_relative _input-check.html id="interested-subscription" label="Gruntwork Subscription" small_text="Includes access to: Infrastructure as Code Library, DevOps Training Library" %}
         {% include_relative _input-check.html id="interested-pro-support" label="Pro Support" %}
         {% include_relative _input-check.html id="interested-ref-arch" label="Reference Architecture" %}
-        {% include_relative _input-check.html id="interested-devops-bootcamp" label="DevOps Bootcamp" %}
```

Comment:
I guess we're also not offering the bootcamp right?

Reply:
Yes, we removed this from the new contact form as well.